### PR TITLE
Release .NET 7 runtime images to microsoft staging MCR

### PR DESCRIPTION
**Pull request by pauld-msft**
Followup to https://github.com/microsoft/Oryx/pull/1643

This PR should officially release the .NET 7 runtime images to the microsoft staging repo.

### CODE REVIEW

**Comment by pauld-msft** [comment]
- [ ] Wait until https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=6901800&view=results runs and publishes to staging before merging this in

File: vsts/pipelines/templates/_buildimageBasesJobTemplate.yml [hunk 1]
```diff
@@ -25,6 +25,8 @@ jobs:
     inputs:
       scriptPath: ${{ parameters.scriptPath }}
       args: ${{ parameters.imageDir }} ${{ parameters.imageDebianFlavor }}
+    env:
+      DOTNET_PRIVATE_STORAGE_ACCOUNT_ACCESS_TOKEN: $(DotnetPrivateStorageAccountAccessToken)
```
Comment (daniv-msft) [inline vsts/pipelines/templates/_buildimageBasesJobTemplate.yml:29]: Hope I didn't miss something: where is this one used? Was it something introduced in the previous PR?

**Review by daniv-msft — APPROVED** [review]

File: vsts/pipelines/templates/_buildimageBasesJobTemplate.yml (hunk 1 above)
Comment (pauld-msft) [reply vsts/pipelines/templates/_buildimageBasesJobTemplate.yml:29]: @daniv-msft Yes it was! This is the environment variable that will be inserted into the staging base Dockerfile as a secret, at this line https://github.com/microsoft/Oryx/blob/main/build/buildRunTimeImageBases.sh#L156

Comment (daniv-msft) [reply vsts/pipelines/templates/_buildimageBasesJobTemplate.yml:29]: Thanks!

**Review by daniv-msft — APPROVED** [review]

**Pull request merged**